npm/builder.js

```diff
@@ -170,44 +168,42 @@ Builder.prototype.prepareRelease = function (releaseVersion, callback) {
   if (process.env.DRY_RUN) {
     log.warn('Dry run! To perform the release, run the command again without DRY_RUN environment variable');
   } else {
-    bfs.updateFileSync('package.json', /"version": "(.*?)"/g, '"version": "' + releaseVersion + '"');
+    this.execSync('npm version ' + releaseVersion);
   }
   callback();
 };
 
-Builder.prototype.commit = function (releaseVersion, callback) {
-  this.execSync('git add -A .');
-  this.execSync('git commit -m "Release ' + releaseVersion + '"');
-  this.execSync('git tag v' + releaseVersion);
-  callback();
-};
-
-Builder.prototype.prepareNextIteration = function (callback) {
-  this.removeDistDirSync();
-  this.execSync('git add -A .');
-  this.execSync('git commit -m "Prepare for next development iteration"');
-  callback();
-};
-
 Builder.prototype.runTest = function (callback) {
   this.execSync('npm run test');
   callback();
 };
 
-Builder.prototype.publish = function (callback) {
-  if (process.env.SKIP_PUBLISH) {
-    log.info('SKIP_PUBLISH environment variable is defined, skipping "publish" task');
-    callback();
-    return;
+Builder.prototype.pushRelease = function () {
+  var result = child_process.execSync('git remote -v').toString('utf8')
```

Comment:
I'm using the following code to get the remote name of the "original" repository.